Drivers/Common/Src/drivers_config.cpp

```diff
@@ -0,0 +1,7 @@
+#include "drivers_config.hpp"
+
+/*
+    creating sbusReceiver instance
+*/
+SBUSReceiver sbus_instance(sbus_uart);
+SBUSReceiver* sbus_pointer = &sbus_instance;
```

Comment:
SBUS is being initialized in drivers_config.cpp, can SM access the pointer from that?

Reply:
I think this will be fixed in the model config PR